comp/core/secrets/go.mod

```diff
@@ -52,6 +52,7 @@ require (
 	github.com/spf13/cobra v1.7.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	go.opentelemetry.io/otel v1.27.0 // indirect
+	go.opentelemetry.io/otel/exporters/prometheus v0.42.0 // indirect
```

Comment:
Is this dependency because we use the Mock module of the telemetry component?

Reply:
Yeah I think it's because I used the `telemetryimpl` in some tests, it's the only import I added